torch/onnx/symbolic_opset16.py

```diff
@@ -47,3 +47,39 @@ def grid_sampler(g, input, grid, mode_enum, padding_mode_enum, align_corners):
         mode_s=mode_s,
         padding_mode_s=padding_mode_s,
     )
+
+
+@symbolic_helper.parse_args("v", "i", "v", "v")
+def scatter_add(g, self, dim, index, src):
+    if symbolic_helper.is_caffe2_aten_fallback():
+        return g.at("scatter", self, dim, index, src, overload_name="src")
+
+    if symbolic_helper._get_tensor_sizes(src) != symbolic_helper._get_tensor_sizes(
+        index
+    ):
+        symbolic_helper._unimplemented(
+            "scatter_add", "index should have the same dimensionality as src"
```

Comment:
I would include the sizes in the message

Reply:
I'm thinking of whether to apply a slice to `src` to make its length equal to `index` when `src` has a longer length. This way matches the behavior of `torch.scatter_add`, and we can then remove this warning. Anyway, adding values to the message is a good suggestion.

Reply:
I decided to leave it as because making `src` align with `index` along all the dimensions would possibly add too many intermediate ops. I've added size values to the message.